stock_prices/stock_prices.py

```diff
@@ -3,7 +3,15 @@
 import argparse
 
 def find_max_profit(prices):
```

Comment:
Nice first pass solution for this one, @nia-ja! 👍 
<img width="1920" alt="Screen Shot 2020-03-05 at 8 40 18 PM" src="https://user-images.githubusercontent.com/16110931/76042046-b6950380-5f21-11ea-91de-d0eb80e3de2e.png">
How could you optimize the runtime complexity of your first pass solution?